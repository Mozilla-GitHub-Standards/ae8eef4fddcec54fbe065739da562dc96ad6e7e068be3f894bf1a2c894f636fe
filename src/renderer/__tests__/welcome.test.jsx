/** @format */

import React from 'react'
import Renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import Welcome from '../pages/welcome'

test('Page: Welcome', () => {
  const component = Renderer.create(
    <MemoryRouter>
      <Welcome />
    </MemoryRouter>
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
