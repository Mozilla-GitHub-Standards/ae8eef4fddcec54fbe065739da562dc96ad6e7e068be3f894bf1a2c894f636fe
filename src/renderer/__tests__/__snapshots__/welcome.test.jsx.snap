// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page: Welcome 1`] = `
<section
  className="welcome__Wrapper-sc-1aduiiu-1 iwiozQ"
>
  <h1
    className="welcome__Title-sc-1aduiiu-0 hGWJmu"
  >
    <a
      href="/about"
      id="start"
      onClick={[Function]}
    >
      About
    </a>
  </h1>
</section>
`;
